lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1003,23 +1007,44 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
     return BitWidth;
   }
 
+  /// Collect the type parameters of a bound generic type. This is needed to
+  /// anchor any typedefs that may appear in parameters so they can be
+  /// resolved in the debugger without needing to query the Swift module.
+  llvm::DINodeArray collectGenericParams(BoundGenericType *BGT) {
+      SmallVector<llvm::Metadata *, 16> TemplateParams;
+      for (auto Param : BGT->getGenericArgs()) {
+        TemplateParams.push_back(DBuilder.createTemplateTypeParameter(
+            TheCU, "", getOrCreateType(DebugTypeInfo::getForwardDecl(Param))));
```

Comment:
On `master-next` `createTemplateTypeParameter` has a fourth parameter `IsDefault` (added in https://reviews.llvm.org/D73462 / https://github.com/apple/llvm-project/commit/7a42babeb83e3927e89e72a0e7e45be9d41b6c23).

What's the right value to pass there to fix the `master-next` build?

Reply:
`false`. Sorry!

Reply:
No worries. Thanks for the confirmation. :-)